src/cpp/core/r_util/RSourceIndex.cpp

```diff
@@ -365,6 +380,15 @@ RSourceIndex::RSourceIndex(const std::string& context,
          tokenOffset = idToken.offset();
       }
       
+      // is this a call to 'shinyServer' or 'shinyUI'?
+      else if (token.contentEquals(L"shinyServer") ||
```

Comment:
I believe that shinyUI is now optional (you can just have a plain e.g. "fluidPage"). Do we need to also infer Shiny based on filenames?  (sort of like the code I sent a link to for detecting the "shiny" extended file type)